{"uid":"1be139b2d9c76169","name":"test_add_contact_emailAddress_null","fullName":"test.Contactos.test_add_contact#test_add_contact_emailAddress_null","historyId":"8e4629bac0ad2265227cdbcb004d5129","time":{"start":1726762233191,"stop":1726762233191,"duration":0},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nTypeError: PayloadContact.build_payload_add_contact() missing 1 required positional argument: 'firstName'","statusTrace":"setup_add_contact = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, [])\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_add_contact_emailAddress_null(setup_add_contact):\n        headers, created_contacts = setup_add_contact\n>       payload = PayloadContact().build_payload_add_contact(salutationName=\"Ms.\",\n                                                             lastName=\"Towel\",\n                                                             name=\"Mike Towel\", emailAddress=\"one@on\",\n                                                             phoneNumber=\"+17945654564\",\n                                                             addressPostalCode=\"0000\",\n                                                             description=\"prueba\")\nE       TypeError: PayloadContact.build_payload_add_contact() missing 1 required positional argument: 'firstName'\n\ntest/Contactos/test_add_contact.py:159: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726762231405,"stop":1726762231405,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_add_contact","time":{"start":1726762233191,"stop":1726762233191,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_add_contact::0","time":{"start":1726762233195,"stop":1726762233195,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Contactos"},{"name":"suite","value":"test_add_contact"},{"name":"host","value":"fv-az529-703"},{"name":"thread","value":"2160-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Contactos.test_add_contact"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"1be139b2d9c76169.json","parameterValues":[]}